lib/platform/bitbucket-server/index.ts

```diff
@@ -901,7 +904,30 @@ export async function updatePr({
       }
     );
 
-    updatePrVersion(prNo, body.version);
+    updatePrVersion(prNo, updatedPr.version);
+
+    const currentState = updatedPr.state;
+    const newState = {
+      [PrState.Open]: 'OPEN',
+      [PrState.Closed]: 'DECLINED',
+    }[state];
+
+    /* TODO: write test for this branch */
+    /* istanbul ignore if */
```

Comment:
Are you sure? Looks like there is a test. 🤔

Reply:
Thanks!